packages/vite/src/node/plugins/html.ts

```diff
@@ -90,6 +96,10 @@ export function addToHTMLProxyCache(
   htmlProxyMap.get(config)!.get(filePath)![index] = code
 }
 
+export function addToHTMLProxyTransformResult(hash: string, code: string) {
```

Comment:
This may need an export type

```suggestion
export function addToHTMLProxyTransformResult(hash: string, code: string): void {
```